app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -55,21 +55,23 @@ interface AddActivityModalProps {
   hasActivityData: boolean;
   inventoryId: string;
   methodology: any;
+  selectedActivity?: SuggestedActivity;
+  referenceNumber: string;
 }
 
 export type Inputs = {
   activity: {
     activityDataAmount?: number | null | undefined;
     activityDataUnit?: string | null | undefined;
-    emissionFactorType: string;
-    co2EmissionFactor: number;
-    n2oEmissionFactor: number;
-    ch4EmissionFactor: number;
+    emissionFactorType?: string;
+    CO2EmissionFactor: number;
+    N2OEmissionFactor: number;
+    CH4EmissionFactor: number;
     dataQuality: string;
     sourceReference: string;
     buildingType: string;
     fuelType: string;
-    totalFuelConsumption: string;
+    totalFuelConsumption?: string | null | undefined;
```

Comment:
Usually you don't need null in here if you provide a default value.
```suggestion
    totalFuelConsumption?: string;
```